src/mmw/js/src/modeling/controllers.js

```diff
@@ -35,22 +36,84 @@ var ModelingController = {
                 name: 'Untitled Project',
                 created_at: Date.now(),
                 area_of_interest: App.map.get('areaOfInterest'),
-                scenarios: new models.ScenariosCollection([
-                    new models.ScenarioModel({
-                        name: 'Current Conditions',
-                        is_current_conditions: true,
-                        active: true
-                    })
-                ])
+                scenarios: new models.ScenariosCollection()
+            });
+
+            // Poll for model results and set the results in the scenario when done.
+            var pollModel = function(scenario) {
+                // If scenario doesn't have its own task model yet, clone and set it from the project.
+                if (!scenario.get('taskModel')) {
+                    scenario.set('taskModel', $.extend(true, {}, project.get('taskModel')));
+                }
+
+                var taskModel = scenario.get('taskModel'),
+                    runoffModel = scenario.get('results').find(function(model) {
```

Comment:
Consider renaming this `runoffResultsModel`, only because `runoffModel` could be confused for TR-55.